### DISCUSSION TITLE
<My Account has been suspendded Ticket ID: 2639845>

### DISCUSSION THREAD

**Discussion by askar-mizo** [community/community#110564] [Discussions]
### Select Topic Area

General

### Body

Dear GitHub Support Team,

I hope this message finds you well. my GitHub account, @mohammad-askar, is still suspended. Unfortunately, I have not received any response regarding the reason for the suspension.

This suspension has led to the deletion of all my projects, causing significant disruption. I kindly request immediate assistance in resolving this matter and understanding the grounds for the account suspension. Your prompt response and support in recovering my projects would be greatly appreciated.
It is very very very important to me.

For your reference, my contact email is [redacted]. I am eager to continue contributing to the GitHub community, and your assistance in this matter is crucial.

Thank you for your attention to this urgent issue.

Sincerely,
Mohammad Askar

**Comment by ettaboyle** [comment]
Hi @askar-mizo! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please refer to your open ticket with them. We cannot help with account related issues and this is not a support portal.

_* original Post updated to remove user email_
